src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENT_PRICE = 40;
+  const DISCOUNT_OVER_6_DAYS = 50;
+  const DISCOUNT_OVER_2_DAYS = 20;
+  let basePrice = days * DAY_RENT_PRICE;
```

Comment:
Prefer using `const` for `basePrice` since it's not intended to be reassigned. Instead, calculate the final price with discounts directly in the `return` statement.

Reply:
Fix this comment

Reply:
> Fix this comment

Not fixed